jbmc/src/java_bytecode/assignments_from_json.cpp

```diff
@@ -199,9 +199,13 @@ static bool has_nondet_length(const jsont &json)
 /// (typed vs. untyped).
 static jsont get_untyped(const jsont &json, const std::string &object_key)
 {
-  if(get_type(json) || has_nondet_length(json))
-    return json[object_key];
-  return json;
+  if(!json.is_object())
```

Comment:
Are you really expecting just any old thing here when it's not an object, or do you expect something specific, such as null for example? If so, check for your expectation.

Reply:
@smowton It could be pretty much anything so I think this change makes sense.